<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="sony-VAIO" name="Default test" tests="5" failures="0" timestamp="15 Sep 2017 09:11:04 GMT" time="20.904" errors="0">
  <testcase name="@AfterTest CloseBrowser" time="0.0" classname="Scenario_Driver_Component.Search_an_item">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:37)
	at Generic_Component.BaseComponent.CloseBrowser(BaseComponent.java:97)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.afterRun(TestRunner.java:1021)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- @AfterTest CloseBrowser -->
  <testcase name="Invalid_Search_Item" time="6.925" classname="Scenario_Driver_Component.Search_an_item"/>
  <testcase name="Invalid_Search_Item" time="1.356" classname="Scenario_Driver_Component.Search_an_item"/>
  <testcase name="Valid_Search_Item" time="1.297" classname="Scenario_Driver_Component.Search_an_item"/>
  <testcase name="Valid_Search_Item" time="2.73" classname="Scenario_Driver_Component.Search_an_item"/>
  <testcase name="Valid_Search_Item" time="1.428" classname="Scenario_Driver_Component.Search_an_item"/>
</testsuite> <!-- Default test -->
